benchmarks/cpp/matmul.cpp

```diff
@@ -61,6 +61,7 @@ void setupMatmul(Fusion* fusion, MmaLayout layout, MatmulParams params) {
 
   fusion->addOutput(d);
 
+  preseg_passes::OptimizationPass<preseg_passes::PreSegmenter>::runPass(fusion);
```

Comment:
This change has been submitted separately as https://github.com/NVIDIA/Fuser/pull/2680.

Reply:
should we change base to main? since #2680 is merged.

surprisingly that didn't happen automatically.

Reply:
wait. I was mistaken. The base of this PR is on the revert.... NVM